crates/egui/src/containers/scroll_area.rs

```diff
@@ -808,20 +809,20 @@ impl Prepared {
                 .frame_state_mut(|state| state.scroll_target[d].take());
 
             if scroll_enabled[d] {
-                delta += if let Some((target_range, align)) = scroll_target {
+                let update = if let Some(target) = scroll_target {
                     let min = content_ui.min_rect().min[d];
```

Comment:
by using destruction you won't forget any fields, even if more are added in the future

```suggestion
                let update = if let Some(target) = scroll_target {
                    let ScrollTarget { ragne, aling, animation } = target;
                    let min = content_ui.min_rect().min[d];
```